### ISSUE TITLE
[MauiEmbedding - Syncfusion] Exception raised on WinUI

### ISSUE THREAD

**Issue by nickrandolph** [unoplatform/uno.extensions#2183]
## Current behavior

Run uno.samples - SyncfusionApp and select the first chart

![image](https://github.com/unoplatform/uno.extensions/assets/1614057/ccfe8e99-570e-4636-b846-a266151ebf80)

## Expected behavior

No exceptions

## How to reproduce it (as minimally and precisely as possible)

Use this branch: https://github.com/unoplatform/Uno.Samples/pull/638
Open and run the Syncfuion sample on Windows
Click on the first of the charts

## Environment

Nuget Package (s):

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] macOS (AppKit)
- [ ] Mac Catalyst
- [ ] Android
- [ ] WebAssembly
- [ ] Windows
- [ ] Skia (WPF)
- [ ] Skia (GTK on Linux/macOS/Windows)
- [ ] Skia (Linux Framebuffer)
- [ ] Build tasks

Visual Studio:

- [ ] 2019 (version: )
- [ ] 2022 (version: )
- [ ] Visual Studio Code (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by agneszitte** [comment]
Same issue when following the Syncfusion documentation https://github.com/unoplatform/uno.extensions/issues/2060

**Comment by SolidRockProgrammer** [comment]
Hi, This issue is a showstopper for us, an idea when it will be fixed?

**Comment by agneszitte** [comment]
@nickrandolph closing this issue as it is a duplicate of https://github.com/unoplatform/uno.extensions-private/issues/200.
We will be able to adjust the documentation + sample app at the same time.
@SolidRockProgrammer here are the details at the moment to unblock you:
https://github.com/unoplatform/uno.extensions/issues/2060#issuecomment-2038233620
